Dockerfile

```diff
@@ -15,7 +20,7 @@ USER electricitybot
 WORKDIR /opt
 
 RUN export PATH="${PATH}":~/.local/bin/ && \
-    python -m pip install --no-cache-dir poetry && \
+    python -m pip install --no-cache-dir poetry==1.2.2 && \
```

Comment:
The same here

Reply:
yep, I missed locking the poetry version too. Changed to `1.1.15`